components/function-controller/config/crd/bases/serverless.kyma-project.io_functions.yaml

```diff
@@ -21,8 +21,9 @@ spec:
   - JSONPath: .metadata.creationTimestamp
     name: Age
     type: date
-  - JSONPath: .status.condition
-    description: Check if the function is ready
+  - JSONPath: .status.phase
```

Comment:
remember to copy this file to resources/fn-controller, along with rbacs file and others that were generated